src/equations/serre_green_naghdi_flat_1d.jl

```diff
@@ -0,0 +1,443 @@
+abstract type AbstractBathymetry end
+struct BathymetryFlat <: AbstractBathymetry end
+"""
+    bathymetry_flat = DispersiveShallowWater.BathymetryFlat()
+
+A singleton struct indicating a flat bathymetry ``b = 0``.
+"""
+const bathymetry_flat = BathymetryFlat()
+
+@doc raw"""
+    SerreGreenNaghdiEquations1D(gravity)
```

Comment:
```suggestion
    SerreGreenNaghdiEquations1D(; gravity_constant, eta0 = 0.0)
```
The leading semicolon is also missing in the docstrings of the other equations. Could you maybe fix that, too?